apps/mobile/src/queries/balance/sip10-balance.query.ts

```diff
@@ -0,0 +1,37 @@
+import { QueryFunctionContext, useQuery } from '@tanstack/react-query';
+
+import { getSip10BalancesService } from '@leather.io/services';
+import { createMoney } from '@leather.io/utils';
+
+export function useSip10AggregateAvailableBalance(addresses: string[]) {
+  const { isLoading, data: balance } = useSip10AggregateBalanceQuery(addresses);
+  return !isLoading && balance ? balance.usd.availableBalance : createMoney(0, 'USD');
+}
+
+export function useSip10AggregateBalanceQuery(addresses: string[]) {
+  return useQuery({
+    queryKey: ['sip10-balances-service-get-sip10-aggregate-balance', addresses],
+    queryFn: ({ signal }: QueryFunctionContext) =>
+      getSip10BalancesService().getSip10AggregateBalance(addresses, signal),
+    refetchOnReconnect: false,
```

Comment:
Maybe we can have a generic query configuration to avoid repeating these props? Then we can over-ride on demand if we need to